source/client/styles/lobby.css

```diff
@@ -91,26 +101,43 @@
 
 #join_game_button {
   border-radius: 0 var(--border-radius) var(--border-radius) 0;
-  margin-left: 0.5rem;
+  margin-left: 1rem;
 }
 
 #join_game_button:before {
   background-image: url(./../assets/images/icons/arrow_icon.svg);
 }
 
+#oracle {
+  position: absolute;
+  width: 15rem;
+  height: 15rem;
+  transform: scaleX(-1);
+  right: 17.5rem;
+  z-index: 1;
+  pointer-events: none;
+}
+
 #lobby_button_wrapper {
   display: flex;
   flex-direction: column;
   align-items: flex-end;
   justify-content: flex-end;
-  gap: 1rem;
+  gap: 2rem;
 }
 
 #start_game_button,
 #practice_game_button {
-  font-size: 1.75rem;
-  width: 15rem;
-  height: 5.75rem;
+  width: 16rem;
+  height: 6.4rem;
+  font-size: 3rem;
+}
+
+#lobby_profile_list_wrapper {
+  position: relative;
+  top: calc(1rem + 30px);
```

Comment:
don't hard code the 30px; px <-> rem conversion isn't constant, so this will probably break on larger-font systems

use display flex or grid instead